buildpipeline/pipeline.json

```diff
@@ -163,6 +202,20 @@
         {
           "Name": "Core-Setup-CrossBuild",
           "Parameters": {
+            "PB_DockerTag": "ubuntu1404_cross_prereqs_v2",
+            "PB_Architecture": "arm",
+            "TargetRid": "linux-arm",
+            "PB_PortableBuild": "-portable"
+          },
+          "ReportingParameters": {
+            "OperatingSystem": "Ubuntu 14.04",
```

Comment:
First, cross build doesn't use the local system headers / libraries , but rather the generated cross file system that contains all the files for the target architecture. And that doesn't need to match the version of the system where the build runs. And second, for ARM, we have decided that the minimum system will be different and it is Ubuntu 14.04. There is no Redhat 7.2 for arm32, as far as I know.